dist/api_data.py

```diff
@@ -1959,7 +1959,17 @@ def __ge__(self, other):
         set the rollback timestamp for the current transaction. This is valid only for prepared
         transactions under the preserve_prepared config. For prepared transactions, a rollback
         timestamp is required, must not be older than the prepare timestamp, and can be set only
-        once. See @ref timestamp_txn_api and @ref timestamp_prepare'''),
+        once. It is ignored if the preserve prepared config is not enabled. See
+        @ref timestamp_txn_api and @ref timestamp_prepare''')
+]),
+
+'WT_SESSION.prepared_id_transaction_uint' : Method([]),
+'WT_SESSION.prepared_id_transaction' : Method([
+    Config('prepared_id', '', r'''
+        set the optional prepared ID for the prepared updates of the current transaction. Multiple
+        transactions can share a prepared transaction ID, as long as they are all guaranteed to
```

Comment:
`prepare transaction ID` to `prepare ID`?

Reply:
done